common/models/key-value-model.js

```diff
@@ -1,36 +1,93 @@
 var g = require('strong-globalize')();
 
+/**
+ * Data model for key-value databases.
+ *
+ * @class KeyValueModel
+ * @inherits {Model}
+ */
+
 module.exports = function(KeyValueModel) {
-  // TODO add api docs
+  /**
+   * Return the value for a given key.
+   *
+   * @param {String} key The key to use when searching.
+   * @options {Object} options
+   * @property
+   * @callback {Function} callback The function to execute upon completion.
+   * @param {Error} err The error object
+   * @promise
+   */
   KeyValueModel.get = function(key, options, callback) {
     throwNotAttached(this.modelName, 'get');
   };
 
-  // TODO add api docs
+  /**
+   * Persist a value using a given key.
+   *
+   * @param {String} key The key to use when searching.
+   * @param {Any} value The value to persist.
+   * @options {Object} options
+   * @property
```

Comment:
IIRC, we support `options.ttl` and it's also possible to pass `ttl` directly as the `options` argument.

```js
KVM.set('key', 'value', 100, cb);
KVM.set('key', 'value', { ttl: 100 }, cb);
```